File: VB/WindowsApplication1/XtraForm1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Skins
Imports DevExpress.LookAndFeel
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Controls

Namespace WindowsApplication1
	Partial Public Class XtraForm1
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub
		Protected Overrides Sub OnLoad(ByVal e As EventArgs)
			MyBase.OnLoad(e)
			InitStylesRadioGroup()
		End Sub
		Private Sub SetMenuManager()
			If testGridControl.MenuManager Is Nothing Then
				testGridControl.MenuManager = barManager1
			Else
				testGridControl.MenuManager = Nothing
			End If
			If testPictureEdit.MenuManager Is Nothing Then
				testPictureEdit.MenuManager = barManager1
			Else
				testPictureEdit.MenuManager = Nothing
			End If
			If btnFix.Text = "Set MenuManager" Then
				btnFix.Text = "Unset MenuManager"
			Else
				btnFix.Text = "Set MenuManager"
			End If
		End Sub
		Private Sub InitStylesRadioGroup()
			For Each skinContainer As SkinContainer In SkinManager.Default.Skins
				rgStyles.Properties.Items.Add(New RadioGroupItem(skinContainer.SkinName, skinContainer.SkinName))
			Next skinContainer
			AddHandler rgStyles.SelectedIndexChanged, AddressOf rgStyles_SelectedIndexChanged
			rgStyles.SelectedIndex = 5
		End Sub
		Private Sub rgStyles_SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs)
			Dim rg As RadioGroup = CType(sender, RadioGroup)
			UserLookAndFeel.Default.SetSkinStyle(rg.Properties.Items(rg.SelectedIndex).Value.ToString())
		End Sub
		Private Sub btnFix_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnFix.Click
			SetMenuManager()
		End Sub
	End Class
End Namespace

File: CS/WindowsApplication1/XtraForm1.cs
using System;
using DevExpress.Skins;
using DevExpress.LookAndFeel;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Controls;

namespace WindowsApplication1 {
    public partial class XtraForm1 : XtraForm {
        public XtraForm1() {
            InitializeComponent();
        }
        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);
            InitStylesRadioGroup();
        }
        private void SetMenuManager() {
            testGridControl.MenuManager = testGridControl.MenuManager == null ? barManager1 : null;
            testPictureEdit.MenuManager = testPictureEdit.MenuManager == null ? barManager1 : null;
            btnFix.Text = btnFix.Text == "Set MenuManager" ? "Unset MenuManager" : "Set MenuManager";
        }
        private void InitStylesRadioGroup() {
            foreach (SkinContainer skinContainer in SkinManager.Default.Skins) {
                rgStyles.Properties.Items.Add(new RadioGroupItem(skinContainer.SkinName, skinContainer.SkinName));
            }
            rgStyles.SelectedIndexChanged += rgStyles_SelectedIndexChanged;
            rgStyles.SelectedIndex = 5;
        }
        private void rgStyles_SelectedIndexChanged(object sender, EventArgs e) {
            RadioGroup rg = (RadioGroup)sender;
            UserLookAndFeel.Default.SetSkinStyle(rg.Properties.Items[rg.SelectedIndex].Value.ToString());
        }
        private void btnFix_Click(object sender, EventArgs e) {
            SetMenuManager();
        }
    }
}